src/actions/actions.js

```diff
@@ -515,3 +517,13 @@ export function saveEnvironment(updateObj, update) {
     update
   };
 }
+
+export function setPreviouslySavedJsmdToCurrentJsmd() {
+  return {
+    type: "SET_PREVIOUSLY_SAVED_JSMD_TO_CURRENT_JSMD"
```

Comment:
could tighten and generalize this to something like `SET_MOST_RECENT_UPLOAD` or `SET_MOST_RECENT_SAVED_CONTENT` -- we may want to compare more than just JSMD (NB title, comments, who knows)